src/iOS/toga_iOS/widgets/scrollcontainer.py

```diff
@@ -0,0 +1,104 @@
+from rubicon.objc import CGSizeMake
+from toga_iOS.libs import (
+    NSLayoutAttributeBottom,
+    NSLayoutAttributeHeight,
+    NSLayoutAttributeLeading,
+    NSLayoutAttributeTop,
+    NSLayoutAttributeTrailing,
+    NSLayoutAttributeWidth,
+    NSLayoutConstraint,
+    NSLayoutRelationEqual,
+    UIScrollView
+)
+from toga_iOS.window import (
+    iOSViewport,
+    UIColor
+)
+from travertino.size import at_least
+from .base import Widget
+
+
+class ScrollContainer(Widget):
+    __current_content = None
+    __scroll_height_constraint = None
+    __scroll_width_constraint = None
+    __vertical_enabled = False
+    __horizontal_enabled = False
+    
+    def __update_content_size(self):
```

Comment:
Why are these "private" methods? The implementation layer isn't surfaced for the user to see, so there's generally no reason to avoid surfacing this method. Worst case, if it's *really* a "don't-mess-with-this" internal detail, single underscore protection is plenty.

Reply:
@freakboy3742 good point, this is mostly out of habit, since it is only needed internally by the refresh/rehint I made it private. But yeah now that I look at it the impl layer can't be accessed externally anyway. I'll make the change and remove it